src/loaders/MaterialLoader.js

```diff
@@ -61,6 +61,7 @@ MaterialLoader.prototype = Object.assign( Object.create( Loader.prototype ), {
 		if ( json.color !== undefined ) material.color.setHex( json.color );
 		if ( json.roughness !== undefined ) material.roughness = json.roughness;
 		if ( json.metalness !== undefined ) material.metalness = json.metalness;
+		if ( json.sheen !== undefined ) material.sheen = (new Color()).setHex( json.sheen );
```

Comment:
Minot style issue:
```js
if ( json.sheen !== undefined ) material.sheen = new Color().setHex( json.sheen );
```

Reply:
I saw this style beign used at some place so i added it